service/worker/indexer/esProcessor.go

```diff
@@ -297,14 +301,14 @@ func newKafkaMessageWithMetrics(kafkaMsg messaging.Message, stopwatch *tally.Sto
 }
 
 func (km *kafkaMessageWithMetrics) Ack() {
-	km.message.Ack()
+	_ = km.message.Ack()
 	if km.swFromAddToAck != nil {
 		km.swFromAddToAck.Stop()
 	}
 }
 
 func (km *kafkaMessageWithMetrics) Nack() {
-	km.message.Nack()
+	_ = km.message.Nack()
```

Comment:
why some of the Nack/Act errors are logged but some not?

Reply:
I was logging opportunistically. If there was easy access to a logger I'd log it, otherwise just explicitly ignored.